host/xtest/Makefile

```diff
@@ -100,16 +100,6 @@ CFLAGS += -I$(OPTEE_PKCS11_TA_HEADERS)
 LDFLAGS += -L$(OPTEE_CLIENT_EXPORT)/lib -lckteec
 endif
 
-ifdef CFG_GP_PACKAGE_PATH
-CFLAGS += -DWITH_GP_TESTS
-
-srcs  += gp_7000.c \
-	 gp_7500.c \
-	 gp_8000.c \
-	 gp_8500.c \
-	 gp_9000.c
-endif
-
```

Comment:
The other `ifdef CFG_GP_PACKAGE_PATH` and the `ifndef CFG_GP_PACKAGE_PATH` below can also be removed.

Reply:
How could I miss them...? Fixed now.